tests/drivers/charger/sbs_charger/testcase.yaml

```diff
@@ -0,0 +1,32 @@
+tests:
+  # section.subsection
+  drivers.sbs_charger.emulated:
+    tags: drivers charger
+    filter: >
+      dt_compat_enabled("sbs,sbs-charger") and
+      (CONFIG_QEMU_TARGET or CONFIG_BOARD_NATIVE_POSIX)
+    extra_args:
+      CONF_FILE="prj.conf;boards/emulated_board.conf"
+      DTC_OVERLAY_FILE="boards/emulated_board.overlay"
+    platform_exclude:
+      qemu_cortex_a53
+      qemu_cortex_a53_smp
+      qemu_kvm_arm64
+      beaglev_starlight_jh7100
+      xenvm
+      xenvm_gicv3
+  drivers.sbs_charger.emulated_64_bit_i2c_addr:
+    tags: drivers charger
+    filter: >
+      dt_compat_enabled("sbs,sbs-charger") and
+      (CONFIG_QEMU_TARGET or CONFIG_BOARD_NATIVE_POSIX)
+    platform_allow:
+      qemu_cortex_a53
+      qemu_cortex_a53_smp
+      qemu_kvm_arm64
+      beaglev_starlight_jh7100
```

Comment:
`beaglev_starlight_jh7100` was removed. We can omit it here.

Reply:
Ack, should alleviate some of the twister failures